libraries/helper.rb

```diff
@@ -43,6 +43,22 @@ def self.older_than_windows2008r2?
         end
       end
 
+      def self.older_than_windows2012?
```

Comment:
I agree. Maybe you can use the os version and check for `<` that version. Perhaps something similar to https://github.com/chef/chef/blob/master/lib/chef/platform/query_helpers.rb#L31